install.sh

```diff
@@ -7,6 +7,21 @@ if [[ -n "$MSYSTEM" ]]; then
   exit 1
 fi
 
+# Check if script was executed with root/sudo
+if [ "$(id -u)" -ne 0 ]; then
+    if [ "$CI" != "true" ]; then
+        echo "You should be using sudo, are you sure you want to continue? [y/n]"
+        read -r answer
+        if [ "$answer" != "${answer#[Yy]}" ] ;then
+            echo "Continuing without sudo. This might cause issues later on."
```

Comment:
I'm thinking of adding something along the lines of "You should know what you're doing as this might cause issues later on". But that might sound a bit demeaning for some people.

Reply:
hmm, into line 16?

Reply:
Yeah but I don't think that's proper 😂

Reply:
`You need to know what you're doing, because this could cause issues later on.` ?